ERCS/erc-7824.md

```diff
@@ -0,0 +1,146 @@
+---
+eip: 7824
+title: ERC for Exit Format in State Channels
+description: Outcomes and asset allocations definition in state channels using the Exit Format.
+author: State Channels (@statechannels), Layer-3 Foundation (@layer-3), Louis Bellet (@mod)
+discussions-to: https://ethereum-magicians.org/c/ercs/57
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-11-22
+---
+
+## Abstract
+
+This ERC defines a standard **Exit Format** for specifying outcomes and asset allocations in state channels. The Exit Format supports diverse token types (native tokens, ERC-20, and others) and accommodates both simple and advanced allocation mechanisms. The standard ensures interoperability and clarity in the distribution of locked assets upon channel finalization, particularly in protocols like Nitro.
+
+## Motivation
+
+State channels rely on clear and interoperable methods for redistributing locked assets when a channel closes. Current implementations lack a standardized format for defining these outcomes, leading to inefficiencies and integration challenges. This ERC introduces a unified **Exit Format** that can be adopted across state channel implementations, improving compatibility and simplifying on-chain interactions.
+
+## Specification
```

Comment:
Your specification doesn't really explain where these structs fit into the big picture, or explain why they need to be standardized separately from the interface that will be using them. Normally we only accept standards that define externally visible behavior (like events/functions.)